modules/administration_manual/pages/configuration/files/file_sharing_configuration.adoc

```diff
@@ -2,9 +2,9 @@
 
 The sharing policy is configured on the Admin page in the *"Sharing"* section.
 
-NOTE: If you don't see the sharing section, try disabling your AdBlock browser plugin.
-      It might also be related to another installed ad blocker in your browser.
-      If so, please disable the plugin and see if that resolves the situation.
+NOTE: If you don't see the sharing section, try disabling your AdBlock browser plugin. +
```

Comment:
Is the `+` necessary here? I don't think so.

Reply:
Yes it is,
It takes the next line as part of the note but in a new line !
AsciiDoc life :smile:

Reply:
If you want each line to render, in HTML, on a separate line, then these are necessary. Otherwise, they're not.